packages/sdks/snippets/react/src/components/NavBar.tsx

```diff
@@ -0,0 +1,17 @@
+import { type BuilderContent } from '@builder.io/sdk-react';
+
+type Link = { url: string; text: string };
+
+export default function NavBar({ links }: { links: BuilderContent }) {
+  return (
+    <nav>
+      <ul>
+        {links.data?.links.map(({ url, text }: Link, index: number) => (
```

Comment:
why not just:
```jsx
<ul>
      {links.data?.links.map(
        (link: {url: string; text: string}, index: number) => (
          <li key={index}>
            <a href={link.url}>{link.text}</a>
          </li>
        ),
      )}
    </ul>
```
that you've done here all of the others: https://github.com/BuilderIO/builder/pull/4019/files#diff-51d94ea8922df7617e711041cfecaac3ebbfa6c43022d9cabe4300bd42029e0dR7

we don't need to add an extra Link type